### ISSUE TITLE
WAI-ARIA 1.1: grid and table

### ISSUE THREAD

**Issue by michael-n-cooper** [w3c/aria#81]
Filed at https://lists.w3.org/Archives/Public/public-pfwg-comments/2015JulSep/0005.html by Sailesh Panchang spanchang02@yahoo.com 

A grid as defined is "An interactive control which contains cells of tabular data arranged in rows and columns, like a table". 
Therefore, it has   a data-table semantic and not semantic of role=none / presentation.
So, does "Grids do not necessarily imply presentation" mean that grids do not imply mere presentational content like a layout table. They _always_ describe relationship.  

Table: Need to clarify or emphasize that this always has a semantic of a data table. 
Because, when one explicitly sets role=table, one cannot set role=none/presentation.  
This is distinct from a native HTML table that can have role=none/presentation.
Thanks,
Sailesh Panchang

**Comment by michael-n-cooper** [comment]
Moving to ARIA 1.2, but we might want to move this to Practices instead.

**Comment by mcking65** [comment]
Michael, We have already finished defining grid practices in APG, and the grid examples are more than half done.

@spanchang, I believe you are   confusing the word "presentation" with the role "presentation" and the concept of a layout table, which is presentational, with the concept of using a grid to group and layout widgets. 

There are many instances of the word presentation in the spec that have nothing to do with the role. When referring to roles, the spec is clear either with markup or context. 

In this particular case, we are talking about a widget. There is actually no such thing as a presentational widget -- it is impossible to make such a thing since only one role can be mapped to an accessibility API. Grid widgets used for layout of other widgets are still all about relationships; they do not have any characteristics of the role presentation.

Are we going to scrub the entire spec for instances of "present", "presenting", and "presentation" to replace them with a different word? I do not think that we should do that. 

I believe this issue should be closed.

**Comment by spanchang** [comment]
Matt,
I request you to re-read my comment of Aug 6, 2015.
Specifically: "Grids do not necessarily imply presentation" should be worded to convey that grids do not imply mere presentational content like a layout table. They always describe relationship.   
Therefore, your conclusion that I am confusing the word "presentation" with the role "presentation" is unfounded.
As a matter of fact, I see my precise concern has been addressed in the W3C Candidate Recommendation 27 October 2016 with an updated definition and explanation for "grid".  
Perhaps you might wish to acknowledge this in the reason for closing the issue.
Updated draft  W3C Candidate Recommendation 27 October 2016

Grid definition: "A composite widget  containing a collection of one or more rows with one or more cells where some or all cells in the grid are focusable by using methods of two-dimensional navigation, such as directional arrow keys".

Explanation: "The grid role does not imply a specific visual, e.g., tabular, presentation. It describes 
relationships among elements. It may be used for purposes as simple as grouping a collection of checkboxes or navigation links or as complex as creating a full-featured spreadsheet application".
This definition and explanation is different from what was in the draft I commented on.
Thanks and regards,
Sailesh Panchang
